.idea/workspace.xml

```diff
@@ -26,11 +26,61 @@
   </component>
```

Comment:
do follow this http://stackoverflow.com/a/1139797/3587928 to properly remove the unnecessary files from git.